mw-webapp/src/component/title/Title.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {Heading} from "@radix-ui/themes";
+import clsx from "clsx";
+import styles from "src/component/title/Title.module.scss";
+
+export enum HeadingLevel {
```

Comment:
add docs for all public members. HeadingLevel is a public member

Reply:
done